tool/colors/update_colors.dart

```diff
@@ -62,8 +71,15 @@ void generateDart(List<String> colors, String colorType, String className) {
 // Generated file - do not edit.
 
 import 'dart:ui';
+''');
+
+  if (colorType == 'css') {
+    buf.writeln("import 'package:css_colors/css_colors.dart';");
+  } else {
+    buf.writeln("import 'package:flutter/src/$colorType/colors.dart';");
+  }
```

Comment:
It might be worth adding an additional arg to this method for `baseUri` to avoid needing this condition here (that way if other packages are added, they're just new calls to this function rather than needing additional tweaks inside it).

Reply:
Much better indeed!